packages/stores/types.ts

```diff
@@ -103,6 +103,7 @@ type TActiveAccount = TAccount & {
     balance?: string | number;
     landing_company_shortcode: 'svg' | 'costarica' | 'maltainvest' | 'malta' | 'iom';
     is_virtual: number;
+    gradient_class: string;
```

Comment:
@hamid-deriv This doesn't seem right, `gradient_class` doesn't exist on `TActiveAccount` 👀

Reply:
I added this because we use `TActiveAccount` as mock data. `@ts-error-expected` is added on the mock data.